config/locales/es/management.yml

```diff
@@ -5,13 +5,26 @@ es:
         unverified_user: Solo se pueden editar cuentas de usuarios verificados
       show:
         title: Cuenta de usuario
+      edit:
+        title: 'Editar cuenta de usuario: Resetear contraseña'
```

Comment:
I think `reestablecer` would be a better choice than `Resetear`. Same at "send_email", "reseted" and "reset_password" strings